components/com_content/src/View/Category/FeedView.php

```diff
@@ -51,10 +51,17 @@ protected function reconcileNames($item)
         $app               = Factory::getApplication();
         $params            = $app->getParams();
         $item->description = '';
-        $obj               = json_decode($item->images);
+        $obj               = json_decode($item->images) ?? new \stdClass();
```

Comment:
You marked Quy's comment about error as resolved but it wasn't...
```suggestion
        $obj               = json_decode($item->images ?? '{}');
```